src/pudl/metadata/resources/eia.py

```diff
@@ -1218,13 +1218,15 @@
     f"out_eia__{freq}_generators": {
         "description": {
             "additional_summary_text": "all generator attributes including calculated capacity factor, heat rate, fuel cost per MMBTU and fuel cost per MWh.",
-            "additional_details_text": f"""This table includes all generators reported to EIA
-with all of the attributes associated with generator records including directly reported,
-imputed and many calculated columns. We suggest using this table if you want to explore
-{freq} attributes about generators and would rather use a more complete and denormalized table.
-Many tables were used to compile this table - if you are more interested in the originally
-reported values, we recommend searching for ``core_eia`` with the column you are most
-interested in.
+            "additional_details_text": f"""This table includes all {freq} attributes
+for all generators reported to EIA-860 and EIA-923.
+
+To provide a complete picture of generator data, this table compiles data from many
+different EIA tables, including directly reported, imputed and calculated columns.
+We suggest using this table if you want to explore {freq} attributes about generators
+and would rather use a more complete and denormalized table. If you are more interested
+in the originally reported values, we recommend searching for core_eia with the column
+you are most interested in.
```

Comment:
This still suggests to me a bit that we've somewhat sinisterly mucked with the data, but I don't have a better proposal for wording.

Reply:
hmm, I can see that. Maybe:

```suggestion
and would rather use a more complete and denormalized table. If your use case requires
precise control over how missingness is addressed, you can retrieve the originally reported
values by searching for core_eia plus the name of the column you are most interested in.
```